### DISCUSSION TITLE
Is the Static Pressure sensor working properly for anyone else?

### DISCUSSION THREAD

**Discussion by missedtheapex** [dahlb/ha_carrier#61] [Q&A]
Before I go through the steps to properly submit an issue, I want to check to see if this is a known issue, expected behavior, or something else.

I have an SYSTXCCITC01-B thermostat running firmware 4.47. All my sensors (and templated helper sensors) are working as expected and updating/reporting as expected...except for the static pressure sensor. It has been stuck at the same value since the integration was first installed. It appears to be fetching the value from the API according to the logs, along with all other values. But contrary to expectations, the pressure value is always the same non-zero value (0.55999... in my case) .  It doesn't vary with the airflow as I would expect, and doesn't go to zero when the fan is off.
<img width="349" alt="Screenshot 2023-10-08 094351" src="https://github.com/dahlb/ha_carrier/assets/27152504/bcaeee2e-e159-4503-927d-676a0641559a">
If I watch the numbers on the thermostat, I can see the reported static pressure change with airflow as expected:
![20231008_120636067_iOS](https://github.com/dahlb/ha_carrier/assets/27152504/c445fd5d-65fb-4820-9680-28f574b8f42b)

in my case, I don't know where the 0.55999 value came from, but it's definitely the only value ever reported in HA. Maybe it's from the initial setup, because it seems to be about the value I'd expect to accompany an absolute max fan speed/air flow in my setup.

Does the Carrier API just not report that value correctly? Is it behaving normally (that is, varying with airflow) for anyone else?

**Comment by dahlb** [comment]
I always assumed it was from the initial setup, and it's the only pressure I've been able to find in the API.  Thanks for letting us know a real time pressure is available from the thermostat though.

**Comment by FSFirenze** [comment]
Same here, the value is stuck at 1.15 inHg

**Comment by Fitz2380** [comment]
My static pressure only changes when I run the airflow verification test under the installation selection in service mode from the ION thermostat.  The thermostat does not appear to have a real time static pressure flow.

**Reply by ohjay73** [reply]
Mine as well.. shows 0.39WC in the stat and 1.76x kpa in home assistant. Was surprised it was reporting in kpa as it should be WC per standard

**Comment by ohjay73** [comment]
even more interesting is the diag from the integration shows the info in WC but the output in the card shows it in kPa. 

"config": {
          "temperature_unit": "C",
          "static_pressure": 0.2566666603088379,
          "mode": "auto",
          "heat_source": "system",
          "limit_min": 10,
          "limit_max": 32,
          "zones": [

![home assistant carrier ](https://github.com/user-attachments/assets/6dd807d0-4816-4bd0-a261-d4578898e566)

**Comment by Fitz2380** [comment]
Yes. The units shown have to be wrong I believe
